target/linux/ramips/base-files/lib/preinit/05_set_iface_mac_ramips

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (C) 2009 OpenWrt.org
+#
+
+preinit_set_mac_address() {
+	. /lib/functions.sh
+	. /lib/functions/system.sh
+
+	case $(board_name) in
+		hc5*61|\
+		hc5661a|\
+		hc5962|\
+		hiwifi,hc5861b)
+			lan_mac=`mtd_get_mac_ascii bdinfo "Vfac_mac "`
```

Comment:
@ynezz 
as you can see, we use this command to get the mac
```
mtd_get_mac_ascii bdinfo "Vfac_mac "
```

So can you see how it's stored on my device(s)?

Reply:
> So can you see how it's stored on my device(s)?

Yes, but I didn't asked about this.

> Could you please provide hexdump of the flash content as it was for example done in https://github.com/openwrt/openwrt/pull/1448#issuecomment-442476695 ? So I can see how it's stored on your device(s).

And I've meant following commands:

```
hexdump -n 64 -C /dev/mtd2 (factory?)
hexdump -C /dev/mtd8 (bdinfo?)
```

So I can see the actual content of the EEPROM memory. Thanks.